src/main/pages/che-7/installation-guide/proc_adding-self-signed-ssl-certificates-to-che.adoc

```diff
@@ -0,0 +1,72 @@
+---
+title: Adding self-signed SSL certificates to Che
+keywords:
+tags: []
+sidebar: che_7_docs
+permalink: che-7/adding-self-signed-ssl-certificates-to-{prod-short}/
+folder: che-7/installation-guide
+summary:
+---
+:page-liquid:
+
+[id="adding-self-signed-ssl-certificates-to-{prod-short}_{context}"]
+= Adding self-signed SSL certificates to {prod-short}
+
+The following procedure describes how to configure {prod-short} to authorize HTTPS communication with various components, such as identity and git servers, by adding information about self-signed SSL certificates to a {prod-short} configuration.
+
+.Prerequisites
+
+* A Kubernetes orchestration tool, the OpenShift command-line tool, `oc`,
+ifeval::["{project-context}" == "che"]
+or the Kubernetes clusters controlling command-line tool, `kubectl`,
```

Comment:
```suggestion
, or the Kubernetes clusters controlling command-line tool, `kubectl`,
```

Reply:
Moving the comma here bc with condiitonals it will render as "...the OpenShift command-line tool, `oc`, is installed"

Reply:
NVM, I'm an idiot :D It looks good.